### ISSUE TITLE
Blade of Eternal Darkness (i:17780) lacks effect

### ISSUE THREAD

**Issue by ifyousayso** [elysium-project/server#1895]
Pre-1.10.0: ".. the proc was a Use ability with a 1.5 second cast and a 30 second cooldown."
https://web.archive.org/web/20050428015706/http://wow.allakhazam.com:80/item.html?witem=17780
Post-1.10.0: Equip: Chance on landing a damaging spell to deal 100 Shadow damage and restore 100 mana to you.

https://wow.gamepedia.com/Blade_of_Eternal_Darkness
WH Comments at http://db.vanillagaming.org/?item=17780
Comments at https://web.archive.org/web/20070124172532/http://wow.allakhazam.com:80/item.html?witem=17780

**Comment by Sabre142** [comment]
This pre-1.10 spell cannot be recreated I believe.

**Comment by ifyousayso** [comment]
That just sound more and more weird..
https://web.archive.org/web/20050302053907/http://wow.allakhazam.com:80/db/spell.html?wspell=21978

**Comment by Phoosy** [comment]
The pre-1.10 spell doesn't even exist in the client anymore as far as I'm aware, hence the statement "This pre-1.10 spell cannot be recreated I believe." I don't believe anyone is going to argue that it wasn't that way back then, but rather it can't be done now.

**Comment by ifyousayso** [comment]
I guess that makes sense.. thanks for clarifying. Too bad non-mana users roll on it now because they don't know it is/will be a caster weapon. :(

**Comment by thepparker** [comment]
Added in a later patch, the previous effect is no longer in the game.

**Comment by samercyn** [comment]
I just had this drop tonight on the Darrowshire realm and decided to equip it to see how it looked. Upon equipping, I noticed the 30 second cooldown on the dagger showed up. Tried to use it, and it worked. Dealt 100 shadow damage and restored 100 mana on a 30 second cooldown and does not trigger the global cooldown. The use effect does not show on the tooltip, but I can confirm that the use effect does actually work.
